common/lib/dependabot/clients/azure.rb

```diff
@@ -183,6 +183,28 @@ def create_pull_request(pr_name, source_branch, target_branch,
           "/_apis/git/repositories/" + source.unscoped_repo +
           "/pullrequests?api-version=5.0", content.to_json)
       end
+
+      def fetch_pull_request(pull_request_id)
```

Comment:
Could rename this to align with the github client:

```suggestion
      def pull_request(pull_request_id)
```